doc/source/reliability/innodb_corrupt_table_action.rst

```diff
@@ -4,7 +4,32 @@
  Handle Corrupted Tables
 =========================
 
-Instead of crashing the server as they used to do, corrupted |InnoDB| tables are simply disabled, so that the database remains available while the corruption is being fixed.
+When a server subsystem tries to access a corrupted table,
+the server may crash.
+If this outcome is not desirable when a corrupted table is encountered,
+set the new system :variable:`innodb_corrupt_table_action` variable
+to a value which allows the ongoing operation to continue
+without crashing the server.
+
+The server error log registers attempts to access corrupted table pages.
+
+.. rubric:: Interacting with the :variable:`innodb_force_recovery` variable
+
+The :variable:`innodb_corrupt_table_action` variable
+may work in conjunction with the :variable:`innodb_force_recovery` variable
```

Comment:
I wonder how :variable:`i_f_r` gets rendered in the output - doesn't it attempt to link to such variable declared in our docs? We don't have one, this should an external link to Oracle doc (like "for more information below" but only a link here)